x/cdp/keeper/fees.go

```diff
@@ -27,6 +27,46 @@ func (k Keeper) CalculateFees(ctx sdk.Context, principal sdk.Coins, periods sdk.
 	return newFees
 }
 
+// UpdateFeesForRiskyCdps calculates fees for risky CDPs
+// The overall logic is first select the CDPs with 10% of the liquidation ratio
+// Then we call calculate fees on each of those CDPs
+// Next we store the result of the fees in the cdp.AccumulatedFees field
+// Finally we set the cdp.FeesUpdated time to the current block time (ctx.BlockTime()) since that
+// is when we made the update
+func (k Keeper) UpdateFeesForRiskyCdps(ctx sdk.Context, collateralDenom string, marketID string) sdk.Error {
+
+	// TODO - QUESTION - should this be modified to get different prices for different assets in future?
```

Comment:
```suggestion
	
```

Comment can be removed -`UpdateFeesForRiskyCdps` is called in a for-loop in the begin blocker for each asset (xrp, btc, etc), so it is already doing this.